src/plugins/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as FastifySwagger from 'fastify-swagger' 
+import * as Fastify from 'fastify'
+
+export  function configureSwagger(fastifyInstance: Fastify.FastifyInstance ) {
```

Comment:
And change this one to:
```suggestion
export function configureSwagger(fastifyInstance: FastifyInstance) {
```